.goreleaser.yaml

```diff
@@ -46,9 +46,7 @@ builds:
           - CGO_ENABLED=0
   # Build CLI binary without embedded UI for linux. Required by our Console.
   - id: plural-cli-console
-    targets:
-      - linux_amd64
-      - linux_arm64
+    targets: [linux_amd64]
```

Comment:
Why was `linux_arm64` removed here?

Reply:
I have noticed that it already builds `linux_arm64` without the UI: https://github.com/pluralsh/plural-cli/pull/410/files#diff-7326b55c062b0f46fe9e39aace0a25f4515cf206040fb91a6fd2cae839f5e826R35-R41

No need to build the same binary twice.